modules/data_release/ajax/AddPermission.php

```diff
@@ -13,24 +13,40 @@
  */
 
 $DB =& Database::singleton();
```

Comment:
`\Database`

Reply:
@paiva It might be nice to use the Suggest feature for small code changes such as the one you suggested.

```suggestion
$DB = \Database::singleton();
```